src/server/test/web/readingsBarMeterQuantity.js

```diff
@@ -124,7 +124,106 @@ mocha.describe('readings API', () => {
 
                 // Add B11 here
 
-                // Add B12 here
+                mocha.it('B12: 1 day bars for 15 minute reading intervals and quantity units with +-inf start/end time & kWh as kg of CO2', async () => {
+                    const unitData = [
+                        {
+                            // u2
+                            name: 'Electric_Utility',
+                            identifier: '',
+                            unitRepresent: Unit.unitRepresentType.QUANTITY,
+                            secInRate: 3600,
+                            typeOfUnit: Unit.unitType.METER,
+                            suffix: '',
+                            displayable: Unit.displayableType.NONE,
+                            preferredDisplay: false,
+                            note: 'special unit'
+                        },
+                        {
+                            // u10
+                            name: 'kg',
+                            identifier: '',
+                            unitRepresent: Unit.unitRepresentType.QUANTITY,
+                            secInRate: 3600,
+                            typeOfUnit: Unit.unitType.UNIT,
+                            suffix: '',
+                            displayable: Unit.displayableType.ALL,
+                            preferredDisplay: false,
+                            note: 'OED created standard unit'
+                        },
+                        {
+                            // u11
```

Comment:
u11 is not specified as needed in the DesignDoc. I commented this out and it ran fine. I think this unit should be removed.

Reply:
Removed u11 from the array.